exasol-s3/src/test/java/com/exasol/spark/s3/S3TableConfTest.java

```diff
@@ -0,0 +1,90 @@
+package com.exasol.spark.s3;
+
+import com.exasol.spark.SparkSessionProvider;
+import com.exasol.spark.common.Option;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.spark.SparkConf;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.types.*;
+import org.apache.spark.sql.util.CaseInsensitiveStringMap;
+import org.junit.jupiter.api.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.containsString;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class S3TableConfTest {
+    private final StructType schema = new StructType(
+            new StructField[] {
+                    new StructField("f_str", DataTypes.StringType, true, Metadata.empty()),
+                    new StructField("f_int", DataTypes.IntegerType, true, Metadata.empty()),
+            }
+    );
+    private final Map<String, String> basic_params = Map.of(Option.TABLE.key(), "some_table");
+    private static SparkSession spark;
+
+    @BeforeEach
+    public void beforeEach() {
+        final SparkConf conf = new SparkConf()
+                .setMaster("local[*]")
+                .set("spark.ui.enabled", "false")
+                .set("spark.driver.host", "localhost");
+        spark = SparkSessionProvider.getSparkSession(conf);
+    }
+
+    @AfterEach
+    public void afterEach() {
+        spark.close();
+    }
+
+    @Test
+    void testOptionsWithoutParams() {
+        final ExasolS3Table s3_table = new ExasolS3Table(schema);
```

Comment:
```suggestion
        final ExasolS3Table s3Table = new ExasolS3Table(schema);
```
We use camelCase for variables